.github/workflows/project-codeflare-release.yml

```diff
@@ -98,7 +98,18 @@ jobs:
     steps:
     - name: Release CodeFlare operator
       run: |
-        gh workflow run tag-and-build.yml --repo ${{ github.event.inputs.codeflare-repository-organization }}/codeflare-operator --ref ${{ github.ref }} --field is-stable=${{ github.event.inputs.is-stable }} --field version=${{ github.event.inputs.operator-version }} --field replaces=${{ github.event.inputs.replaces }} --field codeflare-sdk-version=${{ github.event.inputs.codeflare-sdk-version }} --field appwrapper-version=${{ github.event.inputs.appwrapper-version }} --field kuberay-version=${{ github.event.inputs.kuberay-version }} --field kueue-version=${{ github.event.inputs.kueue-version }} --field quay-organization=${{ github.event.inputs.quay-organization }} --field community-operators-prod-fork-organization=${{ github.event.inputs.codeflare-repository-organization }} --field community-operators-prod-organization=${{ github.event.inputs.community-operators-prod-organization }}
+        gh workflow run tag-and-build.yml --repo ${{ github.event.inputs.codeflare-repository-organization }}/codeflare-operator \
+        --ref ${{ github.ref }} \
+        --field is-stable=${{ github.event.inputs.is-stable }} \
+        --field version=${{ github.event.inputs.operator-version }} \
+        --field replaces=${{ github.event.inputs.replaces }} \
+        --field codeflare-sdk-version=${{ github.event.inputs.codeflare-sdk-version }} \
+        --field appwrapper-version=${{ github.event.inputs.appwrapper-version }} \
+        --field kuberay-version=${{ github.event.inputs.kuberay-version }} \
+        --field kueue-version=${{ github.event.inputs.kueue-version }} \
+        --field quay-organization=${{ github.event.inputs.quay-organization }} \
+        --field community-operators-prod-fork-organization=${{ github.event.inputs.codeflare-repository-organization }} \
+        --field community-operators-prod-organization=${{ github.event.inputs.community-operators-prod-organization }}
```

Comment:
AFAIK this change is not needed, right?

Reply:
It's not, but I left it in to improve readability. The line was extremely long